build/Targets/Packages.props

```diff
@@ -223,6 +223,7 @@
     <VsWebsiteInteropVersion>8.0.0.0-alpha</VsWebsiteInteropVersion>
     <vswhereVersion>1.0.71</vswhereVersion>
     <xunitVersion>2.2.0-beta4-build3444</xunitVersion>
+    <xunitanalyzersVersion>0.5.0</xunitanalyzersVersion>
```

Comment:
@jasonmalinowski No, this was just something I was playing around with on my Mac.